.docker/test-nodes.yml

```diff
@@ -0,0 +1,229 @@
+# Docker Compose file for KDF test nodes
+#
+# Usage:
+#   Start all nodes:     docker compose -f .docker/test-nodes.yml up -d
+#   Start specific:      docker compose -f .docker/test-nodes.yml --profile utxo up -d
+#   Stop all:            docker compose -f .docker/test-nodes.yml down -v
+#   View logs:           docker compose -f .docker/test-nodes.yml logs -f <service>
+#
+# Profiles:
+#   - utxo:   MYCOIN, MYCOIN1 (basic UTXO testing)
+#   - slp:    FORSLP (BCH/SLP token testing)
+#   - qrc20:  QTUM (Qtum/QRC20 testing)
+#   - evm:    GETH (Ethereum/ERC20 testing)
+#   - zombie: ZOMBIE (Zcash-based testing)
+#   - cosmos: NUCLEUS, ATOM, IBC-RELAYER (Tendermint/IBC testing)
+#   - sia:    SIA (Sia testing)
+#
+# Node groups are controlled via compose profiles (see above).
+#
+# For CI/local reuse:
+#   KDF_DOCKER_COMPOSE_ENV=1 - Test harness attaches to running containers
+
+name: kdf-test-nodes
+
+services:
+  # ============================================================================
+  # UTXO Test Nodes
+  # ============================================================================
+
+  mycoin:
+    image: docker.io/gleec/testblockchain:multiarch
+    profiles: ["utxo", "all"]
+    container_name: kdf-mycoin
+    ports:
+      - "8000:8000"
+    environment:
+      - CHAIN=MYCOIN
+      - CLIENTS=2
+      - COIN=Komodo
+      - COIN_RPC_PORT=8000
+      - DAEMON_URL=http://test:test@127.0.0.1:7000
+      - TEST_ADDY=R9imXLs1hEcU9KbFDQq2hJEEJ1P5UoekaF
+      - TEST_WIF=UqqW7f766rADem9heD8vSBvvrdfJb3zg5r8du9rJxPtccjWf7RG9
+      - TEST_PUBKEY=021607076d7a2cb148d542fb9644c04ffc22d2cca752f80755a0402a24c567b17a
+    volumes:
+      - ${ZCASH_PARAMS_PATH:-~/.zcash-params}:/root/.zcash-params:ro
```

Comment:
locally, the utxo node didn't boot up correctly cuz the script fetching the params puts them in `/Users/****/Library/Application Support/ZcashParams` and not in `/Users/****/.zcash-params`.

The export line that `docker-test-nodes-setup.sh` spit out also suggests the latter path.

~i wonder how it passes just fine in macos CI 🤔~, nvm, such docker tests are ran only from a linux parent xD